ansible/group_vars/node.yml

```diff
@@ -60,8 +60,8 @@ nim_waku_websocket_ssl_key: '/etc/letsencrypt/live/{{ nim_waku_websocket_domain
 
 # Waku rln-relay parameters
 nim_waku_rln_relay_dynamic: true
-nim_waku_rln_relay_eth_contract_address: '0xF471d71E9b1455bBF4b85d475afb9BB0954A29c4'
-nim_waku_rln_relay_eth_client_address: 'http://linux-01.ih-eu-mda1.nimbus.sepolia.wg:8556'
+nim_waku_rln_relay_eth_contract_address: '0x4976Df0f61135EF3E5720D92eadE2e5F47A68Ef9'
+nim_waku_rln_relay_eth_client_address: 'https://rpc.cardona.zkevm-rpc.com'
```

Comment:
maybe we want to provide our own alchemy url here, unsure about the rate limits, like @alrevuelta mentioned previously

Reply:
this one is fine by now. if there is any problem we can reconsider.
also since its a public repo, we would need something to protect the API key and don't want to block the deployment.

no action needed.